src/booking/containers/booking-date/booking-date.component.html

```diff
@@ -1,4 +1,18 @@
-<form [formGroup]="formGroup">
+<ng-container *ngIf="isAnyError()">
```

Comment:
This is going to continually call the isAnyError() method every time change detection runs. Can it be turned into a property that can be set on submit?